internal/platform/database/billing/repository_test.go

```
@@ -627,6 +627,61 @@ func TestListFinancialReportSummariesIncludesEmptyLiveMonthWhenAccountExists(t *
 	}
 }
 
+func TestMarkBillOverdueReturnsConcurrentUpdateWhenNoRowsAffected(t *testing.T) {
+	db, mock, repo := newBillingRepoTest(t)
+	defer closeBillingDB(t, db)
+	tx := beginBillingTx(t, db, mock)
+	defer tx.Rollback()
```

golangci-lint (2.11.4)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)